doc/security/security-best-practice-en.md

```diff
@@ -0,0 +1,83 @@
+# Fate security best practice guidance
+
+When the FATE is used to perform tasks such as federated learning, security models and security protocols should be used to ensure that no privacy data is disclosed during the calculation process. This draft attempts to provide security instructions for using FATE.
```

Comment:
Grammar issue, "no privacy data..." should be "no private data.."?

Reply:
private data is more accurate